src/desktop.go

```diff
@@ -183,7 +192,15 @@ func printDesktops(conf *config, desktops []*desktop) {
 		if printSeparator && i > 0 {
 			fmt.Print(dSeparator)
 		}
-		fmt.Printf("[%d] %s", i, v.name)
+		if conf.VerticalSelection {
+			extraIndent := ""
```

Comment:
Shouldn't be this condition active only in case of positive `IndentSelection` or another specific configuration?